frontend/src/app/task-area/task-area.component.ts

```diff
@@ -1,15 +1,47 @@
 import { Component, OnInit } from '@angular/core';
-import { TaskLoaderService, TaskInterface } from '../task-loader.service';
+import { FormControl, FormGroup, Validators } from '@angular/forms';
+import { TaskLoaderService, Task } from '../task-loader.service';
+
 @Component({
   selector: 'app-task-area',
   templateUrl: './task-area.component.html',
   styleUrls: ['./task-area.component.scss'],
 })
 export class TaskAreaComponent implements OnInit {
-  constructor(private readonly taskLoaderService: TaskLoaderService) { }
-  displayedTasks: TaskInterface[] = [];
+  displayedTasks: Task[] = [];
+  currentTab: string = 'All';
+
+  taskText = new FormControl('', [
+    Validators.required,
+    Validators.minLength(4),
+  ]);
+
+  constructor(private readonly taskLoaderService: TaskLoaderService) {}
 
   ngOnInit(): void {
     this.displayedTasks = this.taskLoaderService.getItems();
   }
+
+  checkTask(taskInformation: Task): void {
+    this.displayedTasks = this.taskLoaderService.changeTaskStatus(
+      taskInformation.id,
+      taskInformation.status
+    );
+  }
+
+  createTask(): void {
+    const taskText = this.taskText.value;
```

Comment:
Wired name for form control.

Reply:
done

Reply:
Okay. Now I can see the changes. But I still have a question. Isn't that formControl `new task description field`? Because if I try to understand what is that formControl is by name, I have some misunderstanding because in the task area can be many tasks with editable description in the future. Please, change the name of the form control to something more exact.

Reply:
fix